app/views/branches/_branch_meal.html.erb

```diff
@@ -0,0 +1,30 @@
+<tr class=<%= 'table-success' if branch_meal.active %>>
+  <td><%= branch_meal.id %></td>
+  <td><%= image_tag branch_meal.meal.image, height: 50 if branch_meal.meal.image.attached? %></td>
+  <td><%= branch_meal.meal.name %></td>
+  <td><%= branch_meal.meal.price %></td>
+  <td><%= branch_meal.meal.non_veg ? "Non-Veg" : "Veg" %></td>
+  <td><%= truncate(branch_meal.meal.description, length: 30)%></td>
+  <td><%= branch_meal.available %></td>
+  <td>
+    <div class= "d-flex">
+      <% if branch_meal.active %>
+        <%= link_to toggle_meal_inactive_branch_path(branch, branch_meal_id: branch_meal.id), class: 'btn btn-success btn-sm me-2', role: 'button' do %>
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-eye-fill" viewBox="0 0 16 16">
+            <path d="M10.5 8a2.5 2.5 0 1 1-5 0 2.5 2.5 0 0 1 5 0z"/>
+            <path d="M0 8s3-5.5 8-5.5S16 8 16 8s-3 5.5-8 5.5S0 8 0 8zm8 3.5a3.5 3.5 0 1 0 0-7 3.5 3.5 0 0 0 0 7z"/>
+          </svg>
+        <% end %>
+      <% else %>
+        <%= link_to toggle_meal_active_branch_path(branch, branch_meal_id: branch_meal.id), class: 'btn btn-danger btn-sm me-2', role: 'button' do %>
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-eye-slash-fill" viewBox="0 0 16 16">
+            <path d="m10.79 12.912-1.614-1.615a3.5 3.5 0 0 1-4.474-4.474l-2.06-2.06C.938 6.278 0 8 0 8s3 5.5 8 5.5a7.029 7.029 0 0 0 2.79-.588zM5.21 3.088A7.028 7.028 0 0 1 8 2.5c5 0 8 5.5 8 5.5s-.939 1.721-2.641 3.238l-2.062-2.062a3.5 3.5 0 0 0-4.474-4.474L5.21 3.089z"/>
+            <path d="M5.525 7.646a2.5 2.5 0 0 0 2.829 2.829l-2.83-2.829zm4.95.708-2.829-2.83a2.5 2.5 0 0 1 2.829 2.829zm3.171 6-12-12 .708-.708 12 12-.708.708z"/>
+          </svg>
+        <% end %>
+      <% end %>
+      <%= link_to 'Edit', edit_meal_path(branch_meal.meal), class: 'btn btn-primary btn-sm me-2' %>
```

Comment:
remove edit and delete button from here
Add a message saying this page is just to enable disable meals in a branch
to make changes visit meals page(make this link)

Reply:
SIr delete is for deleting branches_meals. I think it should be present.

Reply:
Done